L/libcgal_julia/build_tarballs.jl

```diff
@@ -88,7 +87,7 @@ dependencies = [
     BuildDependency("MPFR_jll"),
 
     Dependency("CGAL_jll", compat="~5.5"),
```

Comment:
> Please check the discussions in #6653 for details on that MPFR issue, this was a bug in MPFR 4.1.1 and can be avoided by going back to 4.1.0.

This will not work because of CGAL 5.5.2 (see https://github.com/rgcv/libcgal-julia/issues/5, libcgal-julia needs a slight ajustment).

Reply:
``julia
julia> Dependency("CGAL_jll", compat="~5.5")
Dependency(PackageSpec(
  name = CGAL_jll
  version = *
), v"5.5.0", "~5.5", [AnyPlatform], false)

julia> Dependency("CGAL_jll", compat="~5.5.2")
Dependency(PackageSpec(
  name = CGAL_jll
  version = *
), v"5.5.2", "~5.5.2", [AnyPlatform], false)
```

Reply:
I test if CGAl 5.5.2 first fail as it was the case on my local install. `has_null_segment` no more exist in CGAL 5.5.2

Reply:
Not surprisingly, it fails for CGAl 5.5.2 because of errors of compilation of `libcgal-julia/src/straight_skeleton_2.cpp`. 
So the solution I prefer is to fix these errors of compilation to be compatible with the latest stable version CGAL 5.5.2. (also 5.6 is coming soon).  I'll propose a quick fix in some forked repo of libcgal-julia.